pgc_mosaic_query_index.py

```diff
@@ -70,6 +71,8 @@ def main():
                         help="limit search to imagery with both a multispectral and a panchromatic component")
     parser.add_argument("--skip-cmd-txt", action='store_true', default=False,
                         help='Skip writing the txt file containing the input command.')
+    parser.add_argument("--bypass-exclude-list", action='store_true', default=False,
```

Comment:
Can you remove the changes for the mosaic preview tool from this PR? The mosaicker changes belong in their own branch and PR.